pyproject.toml

```diff
@@ -1,3 +1,21 @@
-# Pytest configuration required to prevent pytest from looking for
-# configurations in parent directories. Can be empty.
-[tool.pytest.ini_options]
+[build-system]
+requires = ["setuptools>=45", "setuptools-scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "ts_atbuilding_vents"
+description = "AT dome vent controller for `Vera C. Rubin Observatory <https://lsst.org>`_."
+license = { text = "GPL" }
+urls = { source_code = "https://github.com/lsst-ts/ts_atbuilding_vents"}
+dynamic = ["version"]
+
+[tool.black]
```

Comment:
you should remove these settings for black and isort here. We should rely on ts_pre_commit_config to generate the linting configuration. That allows us to apply project-wide configuration updates. 

Here is what you need to do (after removing these two sessions here), you will have to update the GitHub linting workflow to use workflow in [here](https://github.com/lsst-ts/tssw_workflows).

your workflow should look something like this:

```
name: lint

on: push

jobs:
  call-workflow:
    uses: lsst-ts/tssw_workflows/.github/workflows/lint.yaml@main
```

Reply:
@tribeiro It can be nice to have these in the toml as well, though because it enables lint-as-you-type/save integrations in things like vscode?